lib/tasks/rotate.rake

```diff
@@ -25,12 +24,13 @@ namespace :rotate do
   desc 'encrypt plain OTP secret key'
   task encrypt_otp: :environment do
     num_users = User.where.not(otp_secret_key: nil).count
-    progress = new_progress_bar('Users', num_users)
```

Comment:
Right, I meant but the progress bar is not actually upgrade, so making the progress bar....doesn't do anything?